### PR TITLE
koord-descheduler: add validator and tests for descheduler profile

### PR DESCRIPTION
**Pull request by LeoLiuYan**
### Ⅰ. Describe what this PR does

Add test cases for descheduler profile.

### Ⅱ. Does this pull request fix one issue?

NONE

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

**Comment by eahydra** [comment]
Hello @LeoLiuYan, Nice Work!

File: pkg/descheduler/profile/profile.go [hunk 1]
```diff
@@ -42,12 +43,35 @@ type Map map[string]framework.Handle
...
 		}
+		if err := v.validator(profileCfg); err != nil {
+			return nil, err
+		}
 		m[profileCfg.Name] = p
 	}
 	return m, nil
 }
+
+type cfgValidate struct {
```
Comment (eahydra) [inline pkg/descheduler/profile/profile.go:61]: maybe named `cfgValidator` is better.

**Comment by eahydra** [comment]
Hi @LeoLiuYan , the PR  LGTM. It is recommended to squash all commits into one and re-push after rebase based on main branch.

**Comment by LeoLiuYan** [comment]
> 

OK.  @eahydra

**Review by eahydra — APPROVED** [review]
/lgtm

File: pkg/descheduler/profile/profile_test.go [hunk 2]
```diff
@@ -0,0 +1,186 @@
+/*
```
Comment (jasonliu747) [inline pkg/descheduler/profile/profile_test.go:1]: 这个copyright格式貌似有点问题，咱直接复制其他文件的吧～

**Pull request merged**